### PR TITLE
Prepare release of version 0.4.2

### PR DESCRIPTION
**Pull request by ubergesundheit**

### CODE REVIEW

**Comment by ubergesundheit** [comment]
@piontec could you take a look at the logs of the failing tests? (https://app.circleci.com/pipelines/github/giantswarm/efk-stack-app/244/workflows/f203742f-095c-4ee0-9ec9-77da119095a6/jobs/347). It fails in the `test_pods_available` with `requests.exceptions.ReadTimeout: HTTPSConnectionPool(host='127.0.0.1', port=44063): Read timed out. (read timeout=10)`

**Comment by piontec** [comment]
Crap, hard to tell anything more than you already figured out - it can't connect :| Try to rerun the same test on circleci, but with SSH enabled: login to the instance and check how long does it take for those pods to really be up. It takes around 1 min on my laptop, but maybe the VM there is much more stressed. Also, you can try adding `-d` to `dabs.sh` to enable debug messages (but it won't help much here, as we're getting error from an external test).

**Comment by webwurst** [comment]
@ubergesundheit @piontec Test run through locally with this command:
```bash
export DABS_TAG=0.1.8
./dabs.sh \
 --chart-dir ./helm/efk-stack-app \
 --app-tests-app-config-file ./example_values/kind.yaml \
 --smoke-tests-cluster-type external \
 --functional-tests-cluster-type external \
 --external-cluster-kubeconfig-path kube.config \
 --external-cluster-type external \
 --external-cluster-version "1.18.15" \
 --destination build \
 --debug
```
It looks like on CircleCi an older version is executed:
```
#!/bin/bash -eo pipefail
export DABS_TAG=0.1.7
./dabs.sh -c ./helm/efk-stack-app 
```
Where can this be configured?

**Comment by webwurst** [comment]
Ok, it basically works, just runs into time outs.

**Comment by webwurst** [comment]
Ok, did a rerun with ssh to inspect and it just went through this time.

File: .circleci/config.yml [hunk 1]
```diff
@@ -6,12 +6,22 @@ workflows:
...
           filters:
             tags:
               only: /^v.*/
+
+      - architect/run-tests-with-abs:
+          name: execute chart tests
+          resource_class: 2xlarge
+          chart_dir: ./helm/efk-stack-app
+          # app-build-suite_version: "v0.1.8"
+          app-build-suite_container_tag: "0.1.8"
```
Comment (ubergesundheit) [inline .circleci/config.yml:24]: Don't mind me, just testing something

```suggestion
          app-build-suite_version: "210d995d3e455207dcd0ebdf84152847cdcdae3b"
          app-build-suite_container_tag: "0.1.8-210d995d3e455207dcd0ebdf84152847cdcdae3b"
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -106,9 +110,13 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 ## [0.1.3] 2020-02-10
 - First release
 
+<<<<<<< HEAD
+[Unreleased]: https://github.com/giantswarm/efk-stack-app/compare/v0.3.6...HEAD
+=======
 [Unreleased]: https://github.com/giantswarm/efk-stack-app/compare/v0.4.1...HEAD
 [0.4.1]: https://github.com/giantswarm/efk-stack-app/compare/v0.4.0...v0.4.1
 [0.4.0]: https://github.com/giantswarm/efk-stack-app/compare/v0.3.6...v0.4.0
+>>>>>>> master
```
Comment (ubergesundheit) [inline CHANGELOG.md:119]:
```suggestion
[Unreleased]: https://github.com/giantswarm/efk-stack-app/compare/v0.4.1...HEAD
[0.4.1]: https://github.com/giantswarm/efk-stack-app/compare/v0.4.0...v0.4.1
[0.4.0]: https://github.com/giantswarm/efk-stack-app/compare/v0.3.6...v0.4.0
```

File: helm/efk-stack-app/values.yaml.back [hunk 3]
```diff
@@ -0,0 +1,227 @@
+global:
```
Comment (ubergesundheit) [inline helm/efk-stack-app/values.yaml.back:1]: Delete this file

File: .circleci/config.yml [hunk 4]
```diff
@@ -6,12 +6,22 @@ workflows:
...
+
+      - architect/run-tests-with-abs:
+          name: execute chart tests
+          resource_class: 2xlarge
+          chart_dir: ./helm/efk-stack-app
+          app-build-suite_version: "210d995d3e455207dcd0ebdf84152847cdcdae3b"
+          app-build-suite_container_tag: "0.1.8-210d995d3e455207dcd0ebdf84152847cdcdae3b"
+          filters:
+            tags:
+              only: /^v.*/
```
Comment (ubergesundheit) [inline .circleci/config.yml:27]: Only trigger app after approval

```suggestion
      # Require manual approval for running tests.
      - hold-tests:
          name: hold app tests
          type: approval
          filters:
            # Do not trigger the job on master branch or release tags.
            branches:
              ignore:
                - master
            tags:
              ignore: /^v.*/

      - architect/run-tests-with-abs:
          name: execute chart tests
          requires:
            - hold app tests
            - package and push efk-stack-app chart
          resource_class: 2xlarge
          chart_dir: ./helm/efk-stack-app
          app-build-suite_container_tag: "0.1.8"
          filters:
            # Do not trigger the job on master branch or release tags.
            branches:
              ignore:
                - master
            tags:
              ignore: /^v.*/
```

**Comment by ubergesundheit** [comment]
Looks nice, cannot approve as I am the original PR creator :D

**Pull request merged**